extensions/dacpac/src/telemetry.ts

```diff
@@ -14,5 +14,11 @@ let packageInfo = Utils.getPackageInfo(packageJson);
 export const TelemetryReporter = new AdsTelemetryReporter(packageInfo.name, packageInfo.version, packageInfo.aiKey);
 
 export enum TelemetryViews {
-	SelectOperationPage = 'SelectOperationPage'
+	DataTierApplicationWizard = 'DataTierApplicationWizard',
+	DeployDacpac = 'DeployDacpac',
+	DeployDacpacSettings = 'DeployDacpacSettings',
```

Comment:
I don't see the DeployDacpacSettings telemetry view used anywhere

Reply:
forgot removing this after removing its reference , Thanks for noticing :)